utils/system_utils.py

```diff
@@ -0,0 +1,94 @@
+# Standard Library imports
+import os
+
+def load_preset():
+	# Try to import config.py first
+	try:
+		import config
+	except ModuleNotFoundError:
+		print("config.py not found, exiting.")
+		quit()
+
+	"""
+	Load preset configuration and override default settings if a preset is specified
+	"""
+	if not hasattr(config, 'PRESET') or not config.PRESET:
+		return config
+
+	preset_name = config.PRESET
+	preset_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'presets', preset_name)
```

Comment:
Changing this line to
`preset_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../presets', preset_name)`
should resolve the issue where presets specified in config.py fail to load